pycbc/vetoes/chisq.py

```diff
@@ -329,7 +333,7 @@ def cached_chisq_bins(self, template, psd):
             psd._chisq_cached_key = {}
 
         if not hasattr(template, '_bin_cache'):
-            template._bin_cache = LimitedSizeDict(size_limite=2**2)
```

Comment:
I'm a bit concerned about this change (we've been bitten by caching in places like this a few times in the past year).

Why should the limit be 4? That seems a very odd default to have. 4 is almost 1, and 1 is a much simpler cache size to work with!

I would prefer that this either not be limited, or there be a clear reason here for why we are using 4.

Reply:
Yes, I did ask in pycbc-code why 2**2 .. (i.e. both why 4, and why notate it that way ..)